internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -118,7 +118,16 @@ Resources:
     DependsOn:
     {{- if .ALBListener}}
     {{- range $i, $rule := .ALBListener.Rules }}
-      {{- if $.ALBListener.IsHTTPS }}
+      {{- if $.ImportedALB}}
+        {{- if $.ImportedALB.HasMultipleListeners}}
+      - HTTPSListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
+      - HTTPListenerRedirectRuleForImportedALB
+        {{- else if ne .ImportedALB.HTTPListenerARN ""}}
+      - HTTPListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
+        {{- else if ne .ImportedALB.HTTPSListenerARN ""}}
+      - HTTPSListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
+        {{- end}}
```

Comment:
Is this the same?
```suggestion
        {{- if ne .ImportedALB.HTTPListenerARN ""}}
      - HTTPListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
        {{- if ne .ImportedALB.HTTPSListenerARN ""}}
      - HTTPSListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
        {{- end}}
```

Reply:
don't we need the redirect rule if there are both HTTP and HTTPS?

Reply:
ah missed that one, what about 
```
  {{- if ne .ImportedALB.HTTPListenerARN ""}}
- HTTPListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
  {{- if ne .ImportedALB.HTTPSListenerARN ""}}
- HTTPSListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}
  {{- end}}
  {{- if and (ne .ImportedALB.HTTPListenerARN "") (ne .ImportedALB.HTTPSListenerARN "") }} 
- HTTPListenerRedirectRuleForImportedALB
```

I can see that by having `HasMultipleListeners`, we intend to generalize it so that in the future if/when we support >2 listeners, the same template could still work. However, even with `HasMultipleListeners`, we still need to make a lot of modifications to the CFN templates: we probably need to redirect each of the HTTP listeners to the HTTPS listeners, which means that there would be additional `HTTPListenerRedirectRuleForImportedALB<i>` that wee need to add here.

In this case, I feel like `HasMultipleListeners` doesn't help that much in terms of being future-proof - not to mention that it's probably quite rare that people have > 2 listeners for ALB and we probably will never need to support that.

Reply:
Okay, will make the change-- plus it matches the logic within the template 👍🏼